_rules/iframe-non-empty-title-attribute-cae760.md

```diff
@@ -0,0 +1,208 @@
+
+```html
+<iframe tabindex="-1" src="/test-assets/SC4-1-2-frame-doc.html"> </iframe>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+This `iframe` is not part of [sequential focus navigation][] and has no [content][] because its CSS `display` property is `none`.
+
+```html
+<iframe style="display:none;" src="/test-assets/SC4-1-2-frame-doc.html"></iframe>
+```
+
+#### Inapplicable Example 2
+
+This `iframe` is not part of [sequential focus navigation][] and has no [content][] because its CSS `visibility` property is `hidden`.
+
+```html
+<iframe style="visibility:hidden;" src="/test-assets/SC4-1-2-frame-doc.html"></iframe>
+```
+
+#### Inapplicable Example 3
+
+This `iframe` is not part of [sequential focus navigation][] because its `tabindex` [attribute value][] is `-1`, and has an [explicit role][] of `presentation`.
+
+**Warning:** Using `tabindex="-1"` on an `iframe` makes it, and all its content, inaccessible to keyboard.
+
+```html
+<iframe role="presentation" tabindex="-1" src="/test-assets/SC4-1-2-frame-doc.html"></iframe>
+```
+
+#### Inapplicable Example 4
+
+This `iframe` is not part of [sequential focus navigation][] because its `tabindex` [attribute value][] is `-1`, and has an [explicit role][] of `none`.
+
+**Warning:** Using `tabindex="-1"` on an `iframe` makes it, and all its content, inaccessible to keyboard.
+
+```html
+<iframe role="none" tabindex="-1" src="/test-assets/SC4-1-2-frame-doc.html"></iframe>
+```
+
+#### Inapplicable Example 5
+
+This `iframe` is not part of [sequential focus navigation][] because its `tabindex` [attribute value][] is `-1`, and has an [hidden state][] of `true`.
+
+**Warning:** Using `tabindex="-1"` on an `iframe` makes it, and all its content, inaccessible to keyboard.
+
+```html
+<iframe aria-hidden="true" tabindex="-1" src="/test-assets/SC4-1-2-frame-doc.html"></iframe>
+```
+
+#### Inapplicable Example 6
+
+This `iframe` is not part of [sequential focus navigation][] because its `tabindex` [attribute value][] is `-1`, and has no [content][] inside the `iframe`.
+
+**Warning:** Using `tabindex="-1"` on an `iframe` makes it, and all its content, inaccessible to keyboard.
+
+```html
+<iframe tabindex="-1" src="/test-assets/shared/empty.html"></iframe>
+```
+
+[included in the accessibility tree]: #included-in-the-accessibility-tree 'Definition of included in the accessibility tree'
+[ascii whitespace]: https://infra.spec.whatwg.org/#ascii-whitespace
+[sequential focus navigation]: https://html.spec.whatwg.org/multipage/interaction.html#sequential-focus-navigation 'HTML definition of Sequential focus navigation'
+[user interface component]: https://www.w3.org/TR/WCAG21/#dfn-user-interface-components
+[accessible name computation]: https://www.w3.org/TR/accname
+[explicit role]: #explicit-role
+[hidden state]: #hidden-state
+[content]: https://html.spec.whatwg.org/multipage/dom.html#palpable-content 'HTML definition of Palpable Content'
```

Comment:
I think we might call it "palpable content" everywhere in the rule, given that "content" has already a meaning that is more than just the "palpable content".

Also, "palpable content included in the accessibility tree" is almost [perceivable content](https://github.com/act-rules/act-rules.github.io/blob/rule-cf77f2-bypass-blocks/pages/glossary/perceivable-content.md) I've been working on. I'm not sure whether "perceivable content" would fit your needs here… and/or if we should chisel out a definition that fits both needs…

Reply:
Seeing as this isn't done yet, I'm going to leave this for what it is now. We can always revisit later.

Reply:
I still think we should use "palpable content" all over in the text, rather than "content. Because "content" has another meaning.